src/dataflow-types/src/client/controller.rs

```diff
@@ -221,6 +221,7 @@ where
                                         ),
                                         format!("--processes={}", size_config.scale),
                                         format!("--workers={}", size_config.workers),
+                                        "--log-process-name".to_string(),
```

Comment:
Where is the actual name passed? Shouldn't it be here?

Reply:
Derp, it switched to a bool, not a passed in string.

Nitpick, maybe we should name the arg something like `--should-log-process-name`, so dummies like me don't get confused.